circleci/circleci-pipline-mvn-scan.yaml

```diff
@@ -0,0 +1,33 @@
+version: 2
+executorType: machine
+jobs:
+  build-and-scan:
+    docker:
+      - image: circleci/openjdk:8u171-jdk
+    working_directory: ~/repo
+    environment:
+      MAVEN_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - run:
+          name: maven install
+          command: mvn clean install -DskipTests
+      # Execute WhiteSource scan, curling the latest unified agent. Remeber to set environment variables (api key, url, project name and product name, they need to be upper case too)
+      # WS_APIKEY
+      # WS_WSS_URL
+      # WS_PRODUCTNAME
+      # WS_PROJECTNAME
+      # WS_RESOLVEALLDEPENDENCIES: false
```

Comment:
I would recommend resolving all dependencies and leaving the default.  The recommendation is to delete lines 20-21.

Reply:
@tabac-ws Please update when you get a chance

Reply:
@samq-ws updated.